Panos interface

ivanbojer opened the pull request:
##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
panos_interface

##### ANSIBLE VERSION
<!--- Paste verbatim output from “ansible --version” between quotes below -->
```
ansible 2.1.0.0 
config file = 
configured module search path = Default w/o overrides
```

##### SUMMARY
New panos module that allows Firewall interface configuration. Configure data-port (DP) network interface for DHCP.

Commit:
new module to manage DP interfaces

Commit:
- copyright change for the new module

ivanbojer commented:
ready_for_review

gundalow commented on lib/ansible/modules/network/panos/panos_interface.py:
```diff
@@ -0,0 +1,186 @@
+options:
+    ip_address:
+        description:
+            - IP address (or hostname) of PAN-OS device
```
Capital letters & full stops please

ivanbojer replied:
OK

gundalow commented on lib/ansible/modules/network/panos/panos_interface.py:
```diff
@@ -0,0 +1,186 @@
+'''
+
+EXAMPLES = '''
+# enable DHCP client on ethernet1/1 in zone public
```
This comment should be part of the `- name:`

ivanbojer replied:
OK

Commit:
doc changes as requested

ivanbojer commented:
@gundalow Thank you for the review. I've incorporated all the changes requested. In addition I incorporated changes from the previous comments too (like ANSIBLE_METADATA coming before DOCUMENTATION).

ivanbojer commented:
ready_for_review
